src/components/feed/upload/Category/CategorySelector/index.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { Meta } from '@storybook/react';
+
+import Button from '@/components/common/Button';
+import useModalState from '@/components/common/Modal/useModalState';
+import { default as CategorySelector } from '@/components/feed/upload/Category/CategorySelector';
```

Comment:
p2;
```suggestion
import CategorySelector from '@/components/feed/upload/Category/CategorySelector';
```
가 되는게 좋을 것 같아요.